test/workers/backend_storage_rewrite_worker_test.rb

```diff
@@ -17,7 +17,7 @@ def setup
 
   test '#perform enqueued' do
     Sidekiq::Testing.inline! do
-      Backend::StorageRewrite.expects(:rewrite_provider).with(provider.id)
+      Backend::StorageRewrite.any_instance.expects(:rewrite_provider).with(provider.id)
```

Comment:
This is broken, will be fixed (and new tests added) when the implementation is finalized.